support/svgToPng.js

```diff
@@ -1,38 +1,47 @@
-const fs = require('fs')
-const path = require( 'path' )
-const sharp = require("sharp")
+const fs = require('fs');
+const path = require('path');
+const sharp = require('sharp');
 
 const configFilePath = process.argv[2];
-const outputDir = process.argv[3]
+const outputDir = process.argv[3];
 
-let rawdata = fs.readFileSync(configFilePath)
-let iconsConfig = JSON.parse(rawdata)
+const rawData = fs.readFileSync(configFilePath);
+const iconsConfig = JSON.parse(rawData);
 
-const sourceIconsPath = path.join(path.dirname(configFilePath), '../', iconsConfig.dir)
+const sourceIconsPath = path.join(
+	path.dirname(configFilePath),
+	'..',
+	iconsConfig.dir
+);
 
-for (const icon of iconsConfig.icons) {
-    sharp(`${sourceIconsPath}/${icon.source}.svg`)
-        .resize(16)
+for (const [source, output] of Object.entries(iconsConfig.icons)) {
+	var svgData = fs.readFileSync(`${sourceIconsPath}/${source}.svg`, 'utf8');
+
+	svgData = svgData.replace(/\<svg/g, '<svg fill="#6B6C7E"');
```

Comment:
This is the only brittle part of this whole business. What if the Clay SVGs ever start including a `fill` attribute? You'd get two `fill` attributes then (at the moment, looks like only the "flags" ones do, for obvious reasons.) Why do you use `g` here? I'd expect one svg per file (looking at the Clay SVGs, that's the case for all the ones that we care about here).

Feel free to ship this as-is, but if you want to be a little tighter you could do something like:

```js
svgData = svgData
    .replace(/\s+fill="[^"]+"/, '')
    .replace(/<svg/, '<svg fill="#6b6c7e"');
``` 

(ie. strip `fill` attribute, if any, then add new attribute). I probably wouldn't bother, but mentioning it just for completeness.

Reply:
Roger